costs/anomalies/filters_route.go

```diff
@@ -0,0 +1,185 @@
+package anomalies
+
+import (
+	"database/sql"
+	"encoding/json"
+	"net/http"
+	"reflect"
+	"errors"
+
+	"github.com/trackit/jsonlog"
+
+	"github.com/trackit/trackit-server/db"
+	"github.com/trackit/trackit-server/models"
+	"github.com/trackit/trackit-server/routes"
+	"github.com/trackit/trackit-server/users"
+)
+
+type (
+	// Filter represents a filter.
+	// A filter contains the rule and the associated data.
+	Filter struct {
+		Rule string      `json:"rule" req:"nonzero"`
+		Data interface{} `json:"data" req:"nonzero"`
+	}
+
+	// Filters represents an array of filter.
+	Filters []Filter
+
+	// FiltersBody is the body sent by getAnomaliesFilters
+	// and required by postAnomaliesFilters.
+	FiltersBody struct {
+		Filters Filters `json:"filters" req:"nonzero"`
+	}
+)
+
+var (
+	// availableRules lists all available rules
+	// with their data example.
+	// Examples are used to check the type sent in the body.
+	availableRules = map[string]interface{}{
+		"absolute_date_min": "2019-01-01 00:00:00",
+		"absolute_date_max": "2019-01-01 00:00:00",
```

Comment:
Why not using ISO 8601 format? (e.g. : `2019-01-01T00:00:00Z` )
Existing endpoints that needs to return a date are using this format.